Student sessions

Pokey99 opened the pull request:
Add course list for student sessions.

Pokey99 committed:
Adjust student's courses/list with query
Add mock data for tests
Begin templating student sessions list

Pokey99 committed:
Finish template for student sessions list
Add to testing for student sessions list
Add 401 abort code if not student or teacher

AlaaA0813 commented on portal/templates/courses/list.html:
```diff
@@ -12,24 +12,50 @@
+
+  {% elif user[3] == 'student' %}
+    <h2>Your Schedule</h2>
+
```
too many blank lines

AlaaA0813 commented on portal/templates/index.html:
```diff
@@ -9,10 +9,8 @@
-    <li><a href="/courses/list" class="nav-link">Course List</a></li>
-  {% endif %}
+  <li><a href="/" class="nav-link">Home</a></li>
+  <li><a href="/courses/list" class="nav-link">Course List</a></li>
```
Change Course List to Your Schedule for students

AlaaA0813 commented on tests/test_sessions.py:
```diff
@@ -30,22 +30,21 @@ def test_create_session_student(client, auth):
             with con.cursor() as cur:
                 cur.execute("SELECT * FROM sessions WHERE id = 1")
                 session = cur.fetchone()
+                print(session)
```
get this out of here

AlaaA0813 commented on tests/test_sessions.py:
```diff
@@ -30,22 +30,21 @@ def test_create_session_student(client, auth):
-                cur.execute("SELECT * FROM user_sessions")
+                cur.execute("SELECT * FROM user_sessions WHERE session_id = 1")
                 students = cur.fetchall()
                 print(students)
```
get this out of here too

Pokey99 committed:
Clean up print statements
Clean up empty lines
Revise nav bar for students
Prevent nav bar from showing unless logged in
